src/zope/server/trigger.py

```diff
@@ -158,10 +158,7 @@ def _close(self):
                 self.socket.close()
                 self.socket = None
             if self.trigger is not None:
-                try:
-                    os.close(self.trigger)
-                except OSError:
-                    pass
+                os.close(self.trigger)
```

Comment:
Nice!

Reply:
Thank you coverage.py and coveralls.io, or I wouldn't have dared ;)